src/components/Container/Container.tsx

```diff
@@ -0,0 +1,97 @@
+import styled from "styled-components";
+import { HTMLAttributes } from "react";
+import { Orientation } from "@/components";
+
+type AlignItemsOptions = "start" | "center" | "end";
+type GapOptions = "none" | "xxs" | "xs" | "sm" | "md" | "lg" | "xl" | "xxl";
+type GrowthOptions = "0" | "1" | "2" | "3" | "4" | "5" | "6";
+type JustifyContentOptions =
+  | "flex-start"
+  | "flex-end"
+  | "center"
+  | "space-between"
+  | "space-around"
+  | "space-evenly"
+  | "start"
+  | "end"
+  | "left"
+  | "right";
+type PaddingOptions = "none" | "xxs" | "xs" | "sm" | "md" | "lg" | "xl" | "xxl";
+
+export interface ContainerProps extends HTMLAttributes<HTMLDivElement> {
+  alignItems?: AlignItemsOptions;
+  children?: React.ReactNode;
+  fillWidth?: boolean;
+  gap?: GapOptions;
+  grow?: GrowthOptions;
+  hasBorder?: boolean;
+  isResponsive?: boolean;
+  justifyContent?: JustifyContentOptions;
+  orientation?: Orientation;
+  padding?: PaddingOptions;
+}
+
+const Container = ({
+  alignItems = "center",
+  children,
+  fillWidth = false,
+  gap = "none",
+  grow = "1",
+  hasBorder,
+  isResponsive,
+  justifyContent = "start",
+  orientation = "horizontal",
+  padding = "md",
```

Comment:
Should we have padding none as default

Reply:
Could go either way on this, my gut says md (16px) is likely the most popular (gaps between fields etc).

Reply:
from my experience in what we did in the new unified console we should set it as `none`

Reply:
can we not use fillWidth property and simply use width as optional, with the default as `100%`?
Also, we've added support for extra properties in SQL-Console container like maxWidth, minWidth, we should have them here as well.

Reply:
* I've set the padding default to 'none'
* Added a minWidth and maxWidth props

I'm struggling a bit with having a width prop, seems to be at odds with general flex usage, the last thing we want is people putting in pixel value widths. Seems a lot cleaner to rely on flex grow for the various widths.

Reply:
I am ok with leaving it 100% all the time. And only allowing a maxWidth.

Reply:
Just want to make sure I'm following, if everything has a 100% width, then how can you use `justifyContent="space-between"`. the 100% width overrides `grow="0"`. What's the problem with using auto width and expanding to 100% if we need to? Happy to default to 100% but we probably want to turn that off right? So it's just reversing the default of the current `fillWidth` prop?

Reply:
I guess I'm confused though because having width auto, and having grow='1' by default, seems to achieve the same goal of having the items take up the available space.